Applications/CLI/Tests.cmake

```diff
@@ -371,6 +371,16 @@ if(NOT OGS_USE_MPI)
          DIFF_DATA
          h_us_quad_1000.vtu richards_pcs_0_ts_100_t_100.000000.vtu PRESSURE1 pressure
     )
+     AddTest(
+        NAME LARGE_2D_TwoPhase_PP_Lia_quad
+         PATH Parabolic/TwoPhaseFlowPP/Liakopoulos
+         EXECUTABLE ogs
+         EXECUTABLE_ARGS Twophase_Lia_quad2.prj
+         TESTER vtkdiff
+         ABSTOL 1e-1 RELTOL 1e-1
```

Comment:
You have also the gas_pressure with 1e-8 and 1e-13, and saturation with 1e-14 and 1e-14
absolute and relative tolerances (on my computer -- might be slightly looser on other machines), which you can add to the ctest; Just another lines with 
```
Lia_20.vtu twophaseflow_pcs_0_ts_200_t_20.000000.vtu gas_pressure gas_pressure
```